### ISSUE TITLE
Ensure small screen Navigation is accessible

### ISSUE THREAD

**Issue by magnificode** [Rebuild-Black-Business/RBB-Website#57]
In our PrimaryNav component: https://github.com/Rebuild-Black-Business/RBB-Website/blob/master/src/components/PrimaryNav.js

Primarily I believe we need to add a couple of `aria` roles to the navigation (nav is only toggled on smaller screens).

`aria-controls` should be added to the `<button>` within the `<Box>` component wrapping the Hamburger menu. This should point to the ID of the navigation.

`aria-expanded` should be added to the `<button>` within the `<Box>` and toggled when the button is clicked.

`aria-hidden` should be applied to the `<NavMenu>` component. On desktop this will be set to false, but on mobile this should be toggled when the hamburger nav is clicked.

**Comment by magnificode** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [[PrimaryNav] - Ensure proper small screen a11y](https://trello.com/c/YvFxrW56/53-primarynav-ensure-proper-small-screen-a11y)

**Comment by M0nica** [comment]
I'll start working on this!

**Comment by magnificode** [comment]
Thank you @M0nica !

**Comment by magnificode** [comment]
Handled by #58 - closing!
